.azure/templates/build.yml

```diff
@@ -98,6 +98,12 @@ jobs:
       targetType: inline
       script: tools/prepare-machine.ps1 -ForEbpfBuild -Verbose
 
+  - task: PowerShell@2
```

Comment:
Would an alternative be to add the cxplat project files to the XDP solution?

Reply:
Done